patches/chrome-browser-ui-views-tabs-browser_tab_strip_controller.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc b/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc
+index 6083cd2c41420a6efec778e7ff3af2a74ebb8570..5d1ce2176857603f5364ce4fa9c432d3d44f0ad1 100644
+--- a/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc
++++ b/chrome/browser/ui/views/tabs/browser_tab_strip_controller.cc
+@@ -338,7 +338,7 @@ void BrowserTabStripController::SelectTab(int model_index,
+   // When selecting a split tab, activate the most recently focused tab in the
+   // split.
+   if (tabstrip_->tab_at(model_index)->split().has_value()) {
+-    model_index = GetIndexOfLastFocusedTabInSplit(model_index);
++    // model_index = GetIndexOfLastFocusedTabInSplit(model_index);
```

Comment:
In `cr137`, we could make `BrowserTabStripController::GetIndexOfLastFocusedTabInSplit()` returns passed `model_index` by overriding.
However, `GetIndexOfLastFocusedTabInSplit()` is replaced with util method in `cr138` like below(`split_tabs::GetIndexOfLastActiveTab()`) and it's widely used from multiple places.
If we override here, it should be changed to patching like above in `cr138`. No way to avoid patching to prevent `model_index` update here.

```
  std::optional<split_tabs::SplitTabId> split_id =
      tabstrip_->tab_at(model_index)->split();
  if (split_id.has_value()) {
    model_index = split_tabs::GetIndexOfLastActiveTab(
        browser()->tab_strip_model(), split_id.value());
  }
```